best-practices.md

```diff
@@ -156,6 +157,38 @@ than the overall best resolution.
 - `sar:polarizations` ([sar extension](extensions/sar/)): Provide the polarization content and ordering of a specific asset, similar to `eo:bands`.
 - `sar:product_type` ([sar extension](extensions/sar/)): If mixing multiple product types within a single Item, this can be used to specify the product_type for each asset.
 
+## Working with Media Types
```

Comment:
The headings are a bit strange. The h2 has no content and the h3 is the only header in the h2. This is discouraged in writing...

Reply:
Ah, good catch. I meant to put in more information there on actual practices around using media types, but looks like I forgot to come back and do that. Will work on it now.